package.json

```diff
@@ -28,7 +28,11 @@
     "url": "vslinko/strulidator"
   },
   "scripts": {
-    "prepublish": "babel -d dist lib"
+    "build": "babel -d dist lib",
+    "prepublish": "npm run build",
```

Comment:
I think you could add `npm run lint` before build to ensure code quality before release.